### PR TITLE
devop532 refactor jormungandr coverage tasks, add integration tests

### PR DESCRIPTION
**Pull request by francois-vincent**
devop532

### CODE REVIEW

**Comment by francois-vincent** [comment]
Jenkins, test this please

**Comment by francois-vincent** [comment]
Jenkins, test this please

**Comment by francois-vincent** [comment]
Jenkins, test this please

**Comment by francois-vincent** [comment]
Jenkins, test this please

**Comment by kinnou02** [comment]
there is a conflict

File: README.md [hunk 1]
```diff
@@ -125,11 +125,17 @@ This task has 7 named parameters:
...
-**update_instance**:  deploy a new coverage or deploy it again. (param=coverage name).
+**update_instance**: (param=coverage name) deploy a new coverage or deploy it again.
+
+**update_jormungandr_conf**: deploy jormungandr configuration. Needs a restart_jormungandr to activate (see below).
+
+**deploy_jormungandr_instance_conf**: (param=coverage name) deploy a jormungandr coverage. Needs a restart_jormungandr to activate.
+
+**remove_jormungandr_instance**: (param=coverage name) remove a jormungandr coverage.
+
+**deploy_jormungandr_all_instances_conf**: deploy all jormungandr coverages.
```
Comment (kinnou02) [inline README.md]: It only deploy the configuration of the instances, so the jormungandr configuration is not updated?

Comment (francois-vincent) [inline README.md]: there's another task for that: update_jormungandr_conf

**Pull request merged**